samples/nrf_rpc/protocols_serialization/server/sysbuild.conf

```diff
@@ -0,0 +1,4 @@
+SB_CONFIG_PARTITION_MANAGER=n
```

Comment:
need to know what the plan here with this is, if you disable PM then you disable lots of things, no dfu zip file, none of the signing code in ncs is used (reverts to upstream), no support for ed25519 or kmu, no merged hex file... unless you have a very good reason for doing this and very limited scope of this application, this is very ill-advised